extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
My thought was:
1.  No `<event>` element
2. `<event>` with options
3. `<event>` without options

This would not require any change to your current xsd, all you would have to make sure is to mention that logic in the upcoming documentation PR.